src/Actions/ManagesServers.php

```diff
@@ -44,14 +44,21 @@ public function createServer(array $data, $wait = false, $timeout = 900)
         $response = $this->post('servers', $data);
 
         $server = $response['server'];
-        $server['sudo_password'] = @$response['sudo_password'];
-        $server['database_password'] = @$response['database_password'];
-        $server['provision_command'] = @$response['provision_command'];
+        $initialSudoPassword = @$response['sudo_password'];
```

Comment:
Not a big fan of this syntax.

```suggestion
        $initialSudoPassword = $response["sudo_password"] ?? null;
```

Reply:
Yes, I agree, but I tried to maintain the syntax that was already in use.